cmd/experimental/kjobctl/pkg/builder/ray_cluster_builder_test.go

```diff
@@ -196,7 +197,8 @@ func TestRayClusterBuilder(t *testing.T) {
 				wrappers.MakeRayCluster("", metav1.NamespaceDefault).GenerateName("profile-raycluster-").
 					Annotation("foo", "baz").
 					Label("foo", "bar").
-					Label(constants.ProfileLabel, "profile").
+					Profile("profile").
+					Mode(v1alpha1.RayClusterMode).
```

Comment:
Can we make set `v1alpha1.RayClusterMode` as default in `wrappers.MakeRayCluster`?, (the same goes for the other job types).